src/patternfly/components/ChipGroup/examples/chip-group-toolbar-collapsed-example.hbs

```diff
@@ -1,44 +1,49 @@
 {{#> chip-group chip-group--modifier="pf-m-toolbar" chip-group--id="toolbar-collapsed"}}
-  {{#> chip-group-label}}
+  {{#> chip-group-list-item }}
+    {{#> chip-group-label}}
```

Comment:
With this change the toolbar wraps like this, which is different from what we have visually and in pf-next. I would prefer the label to be part of the <ul> group so that it wraps neater. I haven't caught up with the a11y conversation, so if this isn't the correct way of doing it - thats ok. 
<img width="517" alt="screen shot 2019-01-24 at 10 22 45 am" src="https://user-images.githubusercontent.com/20118816/51688278-397b2900-1fc2-11e9-83a8-2740fcef2f58.png">

Reply:
@christiemolloy ya this is a requirement of Jenn's changes. I'm not sure what we have before was much better tbh. These groups look awkward when they wrap.